apps/site/src/app/admin/participants/components/ParticipantsTable.tsx

```diff
@@ -12,22 +18,96 @@ import { Participant } from "@/lib/admin/useParticipants";
 import ParticipantAction from "./ParticipantAction";
 import RoleBadge from "./RoleBadge";
 
+interface EmptyStateProps {
+	title: string,
+	subtitle?: string,
+	action?: ReactElement,
```

Comment:
tbh not sure if we need `ReactElement` here just to fill a type, taking thoughts